tests/scripts/depends.py

```diff
@@ -381,7 +381,7 @@ def config_symbols_matching(self, regexp):
 
     def __init__(self, options, conf):
         """Gather data about the library and establish a list of domains to test."""
-        build_command = [options.make_command, 'CFLAGS=-Werror']
+        build_command = [options.make_command] + options.make_vars.split(' ') + ['CFLAGS=-Werror']
```

Comment:
+1 to this, not sure on the best fix, but can be done in a follow-up PR - please don't forget!

Reply:
I hadn't noticed that if you omit `--make-vars`, the script doesn't work, because it passes an empty argument to `make`! If I'd noticed, I would have considerd this a blocker.

Also, I hadn't noticed that we also need to change `basic-build-test.sh`. The [nightly CI noticed](https://mbedtls.trustedfirmware.org/job/mbed-tls-nightly-tests/327/execution/node/899/log/).

Fixed in https://github.com/Mbed-TLS/mbedtls/pull/8653 by removing `--make-vars` as well as the need for it.